pkg/operator/etcd_assets/bindata.go

```diff
@@ -1287,11 +1287,13 @@ var _bindata = map[string]func() (*asset, error){
 // directory embedded in the file by go-bindata.
 // For example if you run go-bindata on data/... and data contains the
 // following hierarchy:
-//     data/
-//       foo.txt
-//       img/
-//         a.png
-//         b.png
+//
```

Comment:
that's unfortunately a difference from another go version. Did you run `make update` with go 1.18? The #939 has the same issue bumping to 1.19.

TBH I'm actually fine with you bumping the go.mod and build version to 1.19 here too so we can close the ART PR.

Reply:
awesome, thanks!